deploy/chaos/loadtest/prep.py

```diff
@@ -0,0 +1,73 @@
+"""
+THIS WILL CLEAR YOUR DATABASE AND FILL IT WITH TEST DATA!
+
+Prepare the local database for load testing.
+
+It does the following:
+- Creates an admin user.
+- Creates 1000 channels. Each one has 500 topic nodes. Each node has 30 videos each.
```

Comment:
The current code appears to only create the 1000 channels, it doesn't do the rest yet. Not a big issue, and not a blocker, but just thought I would note it.

Reply:
Ah yeah creating the current setup was enough to get a bad response time for my local machine. Lemme remove the mention of that!